### DISCUSSION TITLE
Building classloader error through pom.xml

### DISCUSSION THREAD

**Discussion by promising-forever** [apache/dolphinscheduler#7427] [Q&A] [answered]
when using resolver.resolvePom()，The io.airlift.resolver framework depends on the version of maven is too low,
maven accessing URL is http://repo.maven.apache.org/maven2，
From 2020.01.15, maven central repository needs https method to access，or it will retrun `501 HTTPS Required`.

**Comment by davidzollo** [comment]
good suggestion
Are you interested in submitting a PR to solve this problem?

BTW,  English is a better choice

**Comment by kezhenxu94** [comment, marked as answer]
We have removed airlift dependencies in recent versions.

**Comment by promising-forever** [comment]
I pull down the latest branch.
Thank you for your answers!
